src/App.js

```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```

Comment:
According to the project requirements, the `App.js` component should be class-based. Please refactor your code from a functional component to a class-based component.

e.g
```
class App extends Component {
  render() {
    return {
      <Calculator />
    };
  }
}
```

Reply:
@KDlamini Thank you for the review , I updated it to a class, but I have a question, If it's not in the project requirements are we supposed to use classes always? because in the react lesson there was a video where the developers of react announced that after the new update using functions is better, to be honest I didn't understood the reason but I am more comfortable in working with functions 😅 
Thank you again , hope you are having a good day 👍🏽

Reply:
Hi @EidHachem

Yes, it is indeed better to use functional components especially because React introduced Hooks which play a similar role to React Lifecycles that can only be used for class-based components. Hooks make managing the state a lot easier. The only reason you are required to use class-based components in this milestone is because Microverse wants you to learn how to use both class-based and functional-based components. You'll refactor your code to functional components in the next projects.

Reply:
@KDlamini Thank you so much for the clarification, I really appreciate it.